Formula/qscintilla2.rb

```diff
@@ -12,14 +12,17 @@ class Qscintilla2 < Formula
   end
 
   option "without-plugin", "Skip building the Qt Designer plugin"
+  option "without-python", "Skip building the python bindings"
 
   depends_on :python => :recommended
```

Comment:
> It's a little counterintuitive, […]

@tdsmith That's extremely counterintuitive and unfortunately doesn't work. Now the default build (with enabled Python support) fails during `make install` of the Python bindings with

```
mkdir: /Library/Python/2.7/site-packages/PyQt4/: Operation not permitted
make: *** [install_pep484_stubs] Error 1
```

because earlier the `python configure.py <...>` step decided:

```
The PEP 484 stub file will be installed in
/Library/Python/2.7/site-packages/PyQt4.
```

Where should this stuff go in case we're not pulling in Homebrew's Python? Should the build system be able to deal with that or do we have to go back to a brewed Python?